### ISSUE TITLE
Line jump calculation does not work correctly for bezier edges

### ISSUE THREAD

**Issue by tortmayr** [eclipse/sprotty#287]
When playing around with the class diagram example I noticed that the line jumping calculation for bezier edges doesn't seem to work as expected:
![weird-routing](https://user-images.githubusercontent.com/2311075/163667625-b29ffb20-7913-461c-92ad-1bb4102d723d.gif)

I guess we should try to fix this and in the meantime disable line jumping for bezier edges. IMO it's better do don't have any line jumps instead of wrongly computed ones.

@planger As the original contributor of the line jumping feature,do yo maybe have any additional insights on what could cause this problem and how to fix it?

**Comment by planger** [comment]
Yes, this is because the [`IntersectionFinder`](https://github.com/eclipse/sprotty/blob/master/packages/sprotty/src/features/edge-intersection/intersection-finder.ts) obtains the routing points of an edge to get its segments. With a [sweepline algorithm](https://github.com/eclipse/sprotty/blob/master/packages/sprotty/src/features/edge-intersection/sweepline.ts) it then finds the intersections among the segments. However, this algorithm assumes that the segments are straight lines.

I'm afraid, it'll be quite some work to compute the intersections among or with bezier edges, as it would potentially require a different algorithm that takes the curves into account.

Imho we should document in the [typedoc](https://github.com/eclipse/sprotty/blob/df4577570c5d95356a4ceb3ecc06754e6a099eec/packages/sprotty/src/features/edge-intersection/intersection-finder.ts#L47) that this doesn't work for bezier edges.

**Comment by spoenemann** [comment]
Right, this was already noted in https://github.com/eclipse/sprotty/pull/245#pullrequestreview-775887389 but we forgot to create a separate issue for it.

**Comment by planger** [comment]
The intersection finder ignores bezier edges with 9241c35. Should we consider this task closed, or do we want to keep it open, in case someone would like to work on supporting line gaps and jumps for bezier lines?
